lib/vm.py

```diff
@@ -524,12 +524,14 @@ def install_uefi_certs(self, auths):
         logging.info(f"Installing UEFI certs to VM {self.uuid}: {[auth.name for auth in auths]}")
         for auth in auths:
             dest = self.host.ssh(['mktemp'])
-            self.host.scp(auth.auth, dest)
-            self.host.ssh([
-                'varstore-set', self.uuid, auth.guid.as_str(), auth.name,
-                str(efi.EFI_AT_ATTRS), dest
-            ])
-            self.host.ssh(['rm', '-f', dest])
+            try:
+                self.host.scp(auth.auth(), dest)
+                self.host.ssh([
+                    'varstore-set', self.uuid, auth.guid.as_str(), auth.name,
+                    str(efi.EFI_AT_ATTRS), dest
+                ])
+            finally:
+                self.host.ssh(['rm', '-f', dest])
```

Comment:
The addition of `try... finally` could be extracted to a separate commit, as it doesn't match the current commit description?